src/main/resources/config.toml

```diff
@@ -0,0 +1,2 @@
+message = 'Click on this message to visit the ServrLink website for information on linking your account' # An alternative message would instruct the user to join the verification server straight away
+url = 'https://servr.link'
```

Comment:
Use double quotes

Reply:
This is how TOML works (At least in TOML's case)